### PR TITLE
Bitwarden enhancements

### PR DESCRIPTION
**Pull request by DavidDeadly**
Primarily, I just wanted to add the 'copy_username' action because it's something I've been needing regularly these days. In the proccess of understanding I found a few things that could be improved, and I just did it. 🤓 The commits have explanations of my decisions.

Initially, I considered keeping this changes local, but I then realized  was the perfect time to help and do my first ever open source contribution. Hope to be helpful!

I'm totally open to debate and feedback, thanks!

### CODE REVIEW

**Review by tomsquest — COMMENTED** [review]
:+1: for the contribution!

Minor styling issue: the "double underscore" could be replace with single underscore, which is traditionally used for private members.
> [pep8] _single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.

**Comment by DavidDeadly** [comment]
Sure!! I was kind of confused about that, but now I know. Thanks!!

**Comment by ManuelSchneid3r** [comment]
@ovitor @tylio what do you think?

File: bitwarden/__init__.py [hunk 1]
```diff
@@ -20,86 +20,35 @@ class Plugin(PluginInstance, TriggerQueryHandler):
 
     def __init__(self):
         TriggerQueryHandler.__init__(self,
-                                     id=md_id,
+                                     id=md_iid,
```
Comment (ManuelSchneid3r) [inline bitwarden/__init__.py:23]: dont use md_iid here. md_id is the id of the plugin, md_iid is the interface id, currently[ v2.1](https://raw.githubusercontent.com/albertlauncher/python/master/albert.pyi)

Comment (DavidDeadly) [reply bitwarden/__init__.py]: Sorry, I thought it was a misspelling because I didn't find where 'md_id' was declared, the linter marked as not defined

**Comment by ManuelSchneid3r** [comment]
please also increase the version.

**Comment by ManuelSchneid3r** [comment]
Actually I meant the plugin version not the interface id. To me it looks like the plugin has some new features. If the UX changes make it 2.0 if it's just some non user recogniceable changes make it 1.4.

**Comment by DavidDeadly** [comment]
Right, my bad... 

Well, I added an action (copy username), that's the only recognisable change in what respects to the user, not sure if that's enough to make a major, please confirm it.

And should I leave the interface id in v2.1?? It's the currently active, isn't it?

**Comment by ManuelSchneid3r** [comment]
Go with 2.0. I dont remember the additions to 2.1 exacty and forgot to document them. Yeah better leave 2.1 in there

**Pull request merged**